tests/test_beatles.py

```diff
@@ -40,18 +40,6 @@ def test_track():
     assert sr == 44100
     assert audio.shape == (44100 * 2,)
 
-    repr_string = (
```

Comment:
why remove these tests instead of keeping them as regression tests of the superclass repr? wouldn't it be nice if the superclass repr could be backwards-compatible? or is that a bunch of extra work?

Reply:
They're definitely not backwards compatible - namely getting the property data types printing exactly the same is basically not possible without huge hacks. The previous repr functions also didn't have consistent implementations. 

The main goal of this PR was to make it ligher weight to contribute a new loader. One of the "annoying" things to add was the repr function + tests, since it's a bit tedious to write.